pkgs/tools/system/mcron/default.nix

```
@@ -1,22 +1,21 @@
-{ fetchurl, lib, stdenv, guile, which, ed, libtool }:
+{ fetchurl, lib, stdenv, guile, pkg-config }:
 
 stdenv.mkDerivation rec {
   pname = "mcron";
-  version = "1.0.6";
+  version = "1.2.1";
 
   src = fetchurl {
     url = "mirror://gnu/mcron/mcron-${version}.tar.gz";
-    sha256 = "0yvrfzzdy2m7fbqkr61fw01wd9r2jpnbyabxhcsfivgxywknl0fy";
+    sha256 = "0bkn235g2ia4f7ispr9d55c7bc18282r3qd8ldhh5q2kiin75zi0";
   };
 
-  patches = [ ./install-vixie-programs.patch ];
-
   # don't attempt to chmod +s files in the nix store
   postPatch = ''
-    substituteInPlace makefile.in --replace "rwxs" "rwx"
+    sed -E -i '/chmod u\+s/d' Makefile.in
   '';
 
-  buildInputs = [ guile which ed libtool ];
+  nativeBuildInputs = [ pkg-config guile ];
```

```suggestion
  nativeBuildInputs = [ pkg-config ];
```
not required in PATH

---

Thanks!